src/components/api.js

```diff
@@ -0,0 +1,25 @@
+const infoAPI = async (city, units = 'metric') => {
+  const apiKey = 'f8584d3f19d9882aec61e1e892543fe7';
+  const response = await fetch(
+    `http://api.openweathermap.org/data/2.5/weather?q=${city}&units=${units}&appid=${apiKey}`,
+  );
+  if (response.status === 200) {
+    const info = await response.json();
```

Comment:
Kindly add a `try {}` and `catch {}` block

Reply:
Done